asserts/confdb.go

```diff
@@ -167,6 +169,67 @@ func (cc *ConfdbControl) Serial() string {
 	return cc.HeaderString("serial")
 }
 
+// ConfdbControlGroup holds a single group in a confdb-control assertion.
+type ConfdbControlGroup struct {
+	Operators       []string
+	Authentications []string
+	Views           []string
+}
+
+// Groups returns the groups in the raw assertion's format.
+func (cc ConfdbControl) Groups() []*ConfdbControlGroup {
```

Comment:
this might need to be dropped and added back later to the intermediate object or should be a helper in confdb taking a []*Operator?